lib/rebirth/base.rb

```diff
@@ -49,6 +54,14 @@ def to_hash
         hash[key] = __get_value(key).map{|v| klass.new(v).to_hash}
       end
 
+      self.class.table_attribute_methods.each do |key, method|
+        if method.respond_to?(:call)
```

Comment:
__[rubocop]__
* Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.